aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -29,16 +29,16 @@ Parameters:
   MinVCPUBatch: 
     Type: Number
     Default: 8
-    MinValue: 2
+    MinValue: 0
     MaxValue: 16
-    AllowedValues: [2,4,8,16]
+    AllowedValues: [0,2,4,8,16]
     Description: 'Maximum VCPUs for Batch Compute Environment [2-16]'
   DesiredVCPUBatch: 
     Type: Number
     Default: 8
-    MinValue: 2
+    MinValue: 0
     MaxValue: 16
-    AllowedValues: [2,4,8,16]
+    AllowedValues: [0,2,4,8,16]
     Description: 'Desired Starting VCPUs for Batch Compute Environment [2-16]'
```

Comment:
The textual description must be updated :)

Reply:
Good catch.  Fixed in a recent commit.